packages/compass-import-export/src/import/import-csv.ts

```diff
@@ -159,12 +151,12 @@ export async function importCSV({
         errorCallback,
       });
 
-      return {
-        ...collectionStream.getStats(),
-        aborted: true,
-      };
+      return makeImportResult(collectionStream, numProcessed, true);
```

Comment:
I didn't think much about what the result of importCSV and importJSON should be. Now it should be the same for both and give us everything we need to update the UI.